src/main/java/org/checkerframework/specimin/UnsolvedClass.java

```diff
@@ -172,7 +188,18 @@ public void updateFieldByType(String currentType, String correctType) {
   public String toString() {
     StringBuilder sb = new StringBuilder();
     sb.append("package ").append(packageName).append(";\n");
-    sb.append("public class ").append(className).append(getTypeVariablesAsString()).append(" {\n");
+    if (isExceptionType) {
+      sb.append("public class ")
+          .append(className)
+          .append(getTypeVariablesAsString())
+          .append(" extends Exception")
+          .append(" {\n");
+    } else {
+      sb.append("public class ")
+          .append(className)
+          .append(getTypeVariablesAsString())
+          .append(" {\n");
```

Comment:
This code is too-duplicated: there should only be one call to `append` for each string. If necessary, you can split this chain of calls up to insert the "extends Exception" in the right place, but you should not duplicate the whole chain.

Reply:
updated

Reply:
rather than leaving a comment like "updated", you can resolve the comment. This has the same semantics (it indicates to the reviewer that you've done what was asked), but it does not generate an email notification and is easier for you. Reviewers generally do not need an email notification about each of their comments, so it is better (and more polite) to save yourself the effort.